src/backend/launcher.ts

```diff
@@ -228,17 +230,27 @@ async function prepareLaunch(
     }
   }
 
+  if (
+    isUmuSupported(gameSettings.wineVersion.type) &&
```

Comment:
This returns `false` if the UMU path does not exist, so it'll never be downloaded

The following patch resolves this issue:
<details>

```patch
Index: src/backend/utils/compatibility_layers.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/backend/utils/compatibility_layers.ts b/src/backend/utils/compatibility_layers.ts
--- a/src/backend/utils/compatibility_layers.ts	(revision Staged)
+++ b/src/backend/utils/compatibility_layers.ts	(date 1721574962214)
@@ -497,11 +497,14 @@
   return defaultUmuPath
 }
 
-export function isUmuSupported(wineType: WineInstallation['type']): boolean {
-  return (
-    wineType === 'proton' &&
-    GlobalConfig.get().getSettings().experimentalFeatures?.umuSupport !==
-      false &&
-    existsSync(getUmuPath())
-  )
+export function isUmuSupported(
+  wineType: WineInstallation['type'],
+  checkUmuInstalled = true
+): boolean {
+  const umuEnabled =
+    GlobalConfig.get().getSettings().experimentalFeatures?.umuSupport !== false
+  const wineVersionSupported = wineType === 'proton'
+  const umuInstalled = checkUmuInstalled ? existsSync(getUmuPath()) : true
+
+  return umuEnabled && wineVersionSupported && umuInstalled
 }
Index: src/backend/launcher.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/backend/launcher.ts b/src/backend/launcher.ts
--- a/src/backend/launcher.ts	(revision Staged)
+++ b/src/backend/launcher.ts	(date 1721575173784)
@@ -231,7 +231,7 @@
   }
 
   if (
-    isUmuSupported(gameSettings.wineVersion.type) &&
+    isUmuSupported(gameSettings.wineVersion.type, false) &&
     !isInstalled('umu') &&
     isOnline() &&
     getUmuPath() === defaultUmuPath

```
</details>